styles.css

```diff
@@ -0,0 +1,189 @@
+body {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 14pt;
+  width: 1280px;
+  margin: 0 auto;
+}
+
+.top-menu {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  color: rgb(255, 255, 255);
+  padding: 0;
+  position: fixed;
+  top: 0;
+  width: 1280px;
+  background-color: rgb(48, 48, 48);
+  margin-top: 0px;
+}
+
+a:hover {
+  background-color: rgb(243, 239, 200);
+}
+
+.top-menu li {
+  margin:  20px;
+}
+
+.top-menu li:first-child {
+  margin-right: auto
+}
+
+ul {
+  list-style-type: none;
+}
+
+.section-header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  color: rgb(32, 32, 32);
+  font-family: 'Montserrat', sans-serif;
+  font-size: 20pt;
+  align-items: center;
+  height: 400px;
+  padding-bottom: 70px;
+  padding-top: 70px;
+
+  border-bottom: 1px solid rgb(190, 190, 190);
+}
+
+.section-header > a {
+  border: 1px solid black;
+  border-radius: 15px;
+  font-size: 12pt;
+  padding: 20px;
+  text-decoration: none;
+  color: black;
+  margin-top: 40px;
+}
+
+.section-quote {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  color: rgb(32, 32, 32);
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-style: italic;
+  align-items: center;
+  height: 250px;
+  padding-bottom: 50px;
+  padding-right: 50px;
+  padding-left: 50px;
+  border-bottom: 1px solid rgb(190, 190, 190);
+
+}
+
+blockquote {
+  quotes: '\201c' '\201d';
+}
+
+blockquote::before {
+  content: open-quote;
+}
+
+blockquote::after {
+  content: close-quote;
+}
+
+.quote-signature {
+  align-self: flex-end;
+}
+
+.section-why {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  color: rgb(32, 32, 32);
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 16pt;
+  align-items: center;
+  height: 400px;
+  border-bottom: 1px solid rgb(190, 190, 190);
+}
+
+.section-why > ul {
+  display: inline-grid; 
+  grid-template-columns: 33% 33% 33%;
+}
+
+.section-why ul > li {
+  display: inline-grid; 
+  font-size: 10pt;
+  grid-template-columns: 33% 33%;
+}
+
+
+
+.section-team {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  color: rgb(32, 32, 32);
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 16pt;
+  align-items: center;
+  height: 600px;
+  border-bottom: 1px solid rgb(190, 190, 190);
+}
+
+.section-team > ul {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  align-content: center;
+  justify-content: space-evenly;  
+  font-size: 12pt;
+}
+
+.section-team ul > li {
+  display: flex;
```

Comment:
There are a few places like this where you could probably dry up your code because it's inheriting things like the display